deploy/21_hlg.ts

```diff
@@ -58,21 +58,23 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
 
   // Future Holograph Utility Token
   const currentNetworkType: NetworkType = network.type;
-  let primaryNetwork: Network;
+  let primaryNetwork: Network | undefined; // Initialize primaryNetwork variable
   if (currentNetworkType == NetworkType.local) {
-    // one billion tokens minted per network on local testing
-    tokenAmount = BigNumber.from('1' + '000' + '000' + '000' + '000000000000000000');
+    // ten billion tokens minted per network on local testnet
+    tokenAmount = BigNumber.from('10' + '000' + '000' + '000' + '000000000000000000');
     primaryNetwork = networks.localhost;
   } else if (currentNetworkType == NetworkType.testnet) {
-    // one hundred million tokens minted per network on testnets
-    tokenAmount = BigNumber.from('100' + '000' + '000' + '000000000000000000');
```

Comment:
This 100 million line was getting overwritten on testnet anyways as can be seen by the amount in the testnet multisig

![image](https://github.com/holographxyz/holograph-protocol/assets/1504794/847f6bf8-a2e6-4abd-b5d2-56adc5fc3931)